prmove.py

```diff
@@ -96,6 +96,12 @@ def get_original_pull_request(self):
         r.raise_for_status()
         self.original_pull_request = r.json()
 
+        mergeable_state = self.original_pull_request['mergeable_state']
+
+        if mergeable_state == 'dirty':
+            raise Exception('Please rebase your branch and update your PR before migrating. '
+                            'Tests will fail for your old PR after rebasing, this is expected and can be ignored.')
```

Comment:
For more information please see http://docs.ansible.com/ansible/repomerge.html#move-issues-and-prs-to-new-repo